.claude/skills/README.md

```diff
@@ -0,0 +1,380 @@
+# Twelve-Factor C.R.A.F.T. Skills System
+
+Comprehensive twelve-factor methodology skills built on the **C.R.A.F.T. framework** (Context-aware, Reusable, Automated, Fast, Tested).
+
+## 🏗️ Architecture
+
+```
+.claude/skills/
+├── twelve-factor-methodology/  # Core methodology & reference
+├── twelve-factor-audit/        # Compliance auditing
+├── twelve-factor-remediation/  # Automated fixing
+├── twelve-factor-monitor/      # Continuous monitoring
+└── skill-generator/            # Meta-skill for creating new skills
+
+context/
+├── bus/shared-context.jsonl    # Shared data bus
+├── events.log                  # Event stream
+├── skills.index.json           # Skill registry
+└── docs/INDEX.md               # Documentation index
+```
+
+## 🚀 Quick Start
+
+### 1. Run Self-Test
+```bash
+cd .claude/skills/twelve-factor-methodology
+./scripts/self-test.sh
+```
+
+### 2. Test Context Bus
+```bash
+./scripts/bus.sh selftest
+```
+
+### 3. Run Full Audit
+```bash
+./scripts/skill-run.sh analyze
+```
+
+## 📦 Available Skills
+
+### twelve-factor-methodology
+**Purpose**: Core reference implementation and analysis
+
+**Commands**:
+```bash
+./scripts/skill-run.sh analyze    # Full compliance analysis
```

Comment:
This seems to be invoked differently than the other scripts, but the script _also_ does not exist.